cloud_build/verify-provenance.sh

```diff
@@ -0,0 +1,34 @@
+#!/bin/bash
+# Copyright 2023 The Flutter Authors. All rights reserved.
+# Use of this source code is governed by a BSD-style license that can be
+# found in the LICENSE file.
+
+set -e
+
+# Download the jq binary in order to obtain the artifact registry url from the
+# docker image provenance.
+echo "Installing jq using curl..."
+curl -Lo jq \
```

Comment:
Is there any other way to install jq? piping something from the internet to bash is considered a security bad practice.

Reply:
That's fair. Perhaps there is already an image out there with jq installed where it wouldn't need to be downloaded at all? I'll look for an alternative.

Reply:
I've updated the installation to be done through `apt`, and specified a version tag as well, since there isn't a hash option for apt.